### DISCUSSION TITLE
Why does Exists Mode for QueryParameters not support empty parameters?

### DISCUSSION THREAD

**Discussion by ppv-milestone** [dotnet/yarp#2553] [Q&A] [answered]
We need to match queries to an interface that accepts a request like this:
http://path/component?hello
But according to documentation the "Exists" Mode in QueryParameters will only match against non-empty parameters and our tests show that Prefix, Exact and Contains seem to be following the same (providing "" as a value to match for causes the service to fail).
Also tried with regular expressions in Path, but those does not seem to match on parameters.
Anyone has a workaround for this? Or maybe supporting empty parameters for Exists could be supported in next version?
Thanks a lot in advance!
Br,
Peter

**Comment by benjaminpetit** [comment, marked as answer]
I don't think that's possible today, but it seems that it could be a valuable addition. Could you open an issue for that?
